### PR TITLE
Logo and readme

### PR DESCRIPTION
**Pull request by nunojesus**

### CODE REVIEW

File: logo/files [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (cornerman) [inline logo/files]: What is this file "files" about?

Comment (nunojesus) [reply logo/files]: I had to create a folder to upload the logo files, but I forget to delete it, sorry again :)

Comment (cornerman) [reply logo/files]: No worries :)

**Comment by cornerman** [comment]
Wow, the readme looks so much nicer now. Thank you for making the world of developers a bit brighter when researching libraries and reading readmes :+1:

**Comment by nunojesus** [comment]
Im happy that you like it :)

Do you need more help? I was looking to your repos, do you need logotypes for this projects: sloth; mycelium; scribe; covenant.

Greetings.

**Comment by cornerman** [comment]
Nice, sloth could definitely use a logo, it is about being lazy and letting the library do some repetitive work for you. If you find the time and have an idea, I would be happy to see it.

**Pull request merged**